app/src/main/res/layout/fragment_delete_ingredient.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/delete_title_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="Please select ingredient to delete from the dropdown menu"
```

Comment:
string file
```suggestion
        android:text="Please select ingredient to delete from the dropdown menu"
```

Reply:
done, thanks. I appreciate the ones you corrected.